tools/pip/setup.py

```diff
@@ -28,6 +28,9 @@
 if platform.system() == 'Linux':
     sys.argv.append('--universal')
     sys.argv.append('--plat-name=manylinux1_x86_64')
+if platform.system() == 'Windows':
+    sys.argv.append('--universal')
```

Comment:
Do you have some more insights into why this works?
It's not supported by wheel https://packaging.python.org/guides/distributing-packages-using-setuptools/#universal-wheels

Reply:
@yajiedesign may provide better answer.

@leezu --universal mean can run on both p2 and p3. I don't think we have C extensions. We only use ctype to bind to a share library. Linux pip setup also use "--universal".

Reply:
@leezu because we don't compile C extensions through python.We compile is self and call via ctypes use it.for python this is a pure python package.

Reply:
We do have Cython for C extensions. If we don't use Cython for the pypi build, I agree with you.

Reply:
@leezu windows pip not use Cython.I don't know about Linux,